core/src/style/polygonStyle.cpp

```diff
@@ -65,56 +96,114 @@ void PolygonStyle::constructShaderProgram() {
     m_shaderProgram->setSourceStrings(fragShaderSrcStr, vertShaderSrcStr);
 }
 
-VboMesh* PolygonStyle::newMesh() const {
-    return new Mesh(m_vertexLayout, m_drawMode);
-}
+namespace {
```

Comment:
"PolygonStyle::Builder" would make sense to me - we have to make sure to
distinguish it from "PolygonBuilder" in builders.h too haha
On Jan 29, 2016 6:38 PM, "Hannes Janetzek" <notifications@github.com> wrote:

> In core/src/style/polygonStyle.cpp
> <https://github.com/tangrams/tangram-es/pull/514#discussion_r51330767>:
>
> > @@ -65,56 +96,114 @@ void PolygonStyle::constructShaderProgram() {
> >      m_shaderProgram->setSourceStrings(fragShaderSrcStr, vertShaderSrcStr);
> >  }
> >
> > -VboMesh* PolygonStyle::newMesh() const {
> > -    return new Mesh(m_vertexLayout, m_drawMode);
> > -}
> > +namespace {
>
> Yeah, saw that too - what do you suggest? Using named namespace from
> filename (minus 'Style') or prefix all structs with StyleX ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tangrams/tangram-es/pull/514/files#r51330767>.
>